C/CvxCompress/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "CvxCompress"
+version = v"1.0.0"
+
+# Collection of sources required to build CvxCompress
+sources = [
+    GitSource(
+        "https://github.com/ChevronETC/CvxCompress.git",
+        "55e072862196e917e3ecece81f014e93d60cdf81"
+    )
+]
+
+# Bash recipe for building across platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/CvxCompress
+make -j${nproc}
+mkdir -p ${libdir}
+cp libcvxcompress.so ${libdir}/libcvxcompress.${dlext}
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+  Linux(:x86_64, libc=:glibc)
+]
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libcvxcompress", :libcvxcompress)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = []
```

Comment:
```
┌ Warning: To ensure that the correct version of libgomp is found at runtime, add the following entry to the list of dependencies of this builder
│ 
│     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
└ @ BinaryBuilder.Auditor /depot/packages/BinaryBuilder/TCJai/src/auditor/compiler_abi.jl:4
```
```suggestion
dependencies = [
    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
]
```
You'll also need to load `Pkg` at the beginning of the script